frontend/src/components/404/index.tsx

```diff
@@ -5,6 +5,9 @@ import ErrorComponent from '../common/ErrorPage';
 export default function NotFoundComponent() {
   const { t } = useTranslation();
   return (
-    <ErrorComponent graphic={headlampBrokenImage} title={t(`Whoops! This page doesn't exist`)} />
+    <ErrorComponent
+      graphic={headlampBrokenImage as any}
```

Comment:
headlamp-plugin build step complained about the types here, not a nice fix but the type casting is very local and not that important here